buildspec.yml

```diff
@@ -0,0 +1,26 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.8
```

Comment:
I'm pretty sure the container is running 3.6x - don't know if that is gonna cause any issues?